protocol/contracts/dao/Implementation.sol

```diff
@@ -34,11 +34,7 @@ contract Implementation is State, Bonding, Market, Regulator, Govern {
         // Reward committer
         incentivize(msg.sender, Constants.getAdvanceIncentive());
         // Dev rewards
```

Comment:
nit: should also remove this comment which has no purpose anymore after the code block beow it is gone